trigger/patch.go

```diff
@@ -382,6 +382,16 @@ func (t *patchTriggers) patchFamilyOutcome(sub *event.Subscription) (*notificati
 	if t.event.EventType != event.PatchChildrenCompletion {
 		return nil, nil
 	}
+
+	//Don't notify the user of the patch outcome if they aborted the patch
+	aborted, err := model.IsAborted(t.patch.Id.Hex())
```

Comment:
Question for my own understanding: since this is relying on the `(Version).Aborted` field, does the version aborted field indicate that _any_ task has been aborted, or that _all_ remaining tasks to run are aborted? For example, let's say you abort single running task in the patch, but leave the rest of the ones still running alone. When the remaining tasks finish, will this suppress the notification? I imagine we would only want to suppress patch aborted notifications only in the case where the user aborted all the remaining tasks that aren't done running.

I glanced through [the update logic for version statuses](https://github.com/evergreen-ci/evergreen/blob/84603e68a746dd779161f2a089347761f1cbbb6b/model/task_lifecycle.go#L1257-L1269), and [for build statuses](https://github.com/evergreen-ci/evergreen/blob/299bf20c8da9e353b1097183672cd5169ff3447d/model/task_lifecycle.go#L1148-L1163). It seems like it follow the former definition (i.e. any task abort counts as abort), but I could be wrong.

Reply:
If I understand correctly, it's the latter. 

But I'll test this to make sure and update back here.

Reply:
I can confirm. I aborted a task [here](https://spruce.mongodb.com/version/6399e8e5850e613cd70aeab1/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) and the version's aborted field remained false.